src/games/gateways/games.gateway.ts

```diff
@@ -21,12 +32,17 @@ export class GamesGateway implements OnGatewayInit, OnModuleInit {
     private events: Events,
   ) {}
 
+  async emit(event: WebsocketEvent, payload: Serializable<GameDto>) {
+    this.socket.emit(event, await serialize(payload));
+  }
+
   @WsAuthorized()
   @SubscribeMessage('replace player')
```

Comment:
a bit of a side-issue but maybe in a follow-up we make these more consistant with FE and make an enum for these

Reply:
Enum for what? The event names are already an enum

Reply:
`'replace player'` could be `Events.ReplacePlayer`?

Reply:
It's not an event, though - it's websocket call. Nevertheless, it's a good idea.

Reply:
Ah. Right. WebSocketEvents